templates/CRM/Contact/Form/Contact.tpl

```diff
@@ -185,7 +185,7 @@
         subTypeValues = null;
       }
       else if (!subTypeValues) {
-        subTypeValues = {/literal}{if isset($paramSubType)}"{$paramSubType}"{else}""{/if}{literal};
+        subTypeValues = {/literal}{$paramSubType}{literal};
```

Comment:
This needs the quotes to be valid javascript.

Reply:
Yep- you are right!